docs/conf.py

```diff
@@ -211,13 +211,10 @@
 latex_elements = {
```

Comment:
Formatting in here, that's fine, I never worried too much about formatting in this file, but it's good to have it tidied up.